### PR TITLE
[4.0] Change list to switcher

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
The Rule column has 2 values. Change it from list to switcher.

### Testing Instructions
Go to Global Configuration > Users > Email Domain Options
Click + icon

### Expected result
switcher

### Actual result
list

### CODE REVIEW

**Comment by brianteeman** [comment]
Just for reference it has been stated elsewhere that the switch should be used only when there are opposite value not just because there are only two options

**Comment by Quy** [comment]
Thanks for the reminder. In this case, they are opposite values.

File: administrator/components/com_users/forms/config_domain.xml [hunk 1]
```diff
@@ -11,15 +11,15 @@
 
 		<field
 			name="rule"
-			type="list"
+			type="radio"
 			label="COM_USERS_CONFIG_FIELD_DOMAIN_RULE_LABEL"
 			description="COM_USERS_CONFIG_FIELD_DOMAIN_RULE_DESC"
-			required="true"
```
Comment (SharkyKZ) [inline administrator/components/com_users/forms/config_domain.xml:17]: Revert. Also would be a good idea to add `validate="options"`.

Comment (Quy) [reply administrator/components/com_users/forms/config_domain.xml:17]: Keep filter?

Comment (SharkyKZ) [reply administrator/components/com_users/forms/config_domain.xml:17]: Yes. But it looks like it doesn't work for subforms now.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 1de8d1e8ee8c0be23102fcabedd86a76a9938f94<br /><br />Switcher function works as intended.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27912">issues.joomla.org/tracker/joomla-cms/27912</a>.</sub>

**Comment by brianteeman** [comment]
Sorry this just looks wrong to me - just as it looks wrong in subfields
image

**Comment by Quy** [comment]
Isn't that a separate issue/PR for switcher styling? With switcher, you can easily see which ones are allowed/disallowed.

**Comment by brianteeman** [comment]
Its just a "looks odd" comment regarding having a swticher here and not having a select

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 1de8d1e8ee8c0be23102fcabedd86a76a9938f94<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27912">issues.joomla.org/tracker/joomla-cms/27912</a>.</sub>

**Comment by chmst** [comment]
It looks a bit awkward, this is true, but is better visible than a select (my opinion). 

 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27912">issues.joomla.org/tracker/joomla-cms/27912</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27912">issues.joomla.org/tracker/joomla-cms/27912</a>.</sub>

**Comment by richard67** [comment]
Thanks.

**Pull request merged**